modules/setting-up-cpu-manager.adoc

```diff
@@ -172,8 +179,28 @@ QoS Class:       Guaranteed
 Node-Selectors:  cpumanager=true
 ----
 
-. Verify that the `cgroups` are set up correctly. Get the process ID (PID) of the `pause` process:
+. Verify that a CPU has been exclusively assigned to the pod by running the following command:
++
+[source,terminal]
+----
+# oc describe node --selector='cpumanager=true' | grep -i cpumanager- -B2
+----
++
+.Example output
+[source,terminal]
+----
+NAMESPACE    NAME                CPU Requests  CPU Limits  Memory Requests  Memory Limits  Age
+cpuman       cpumanager-mlrrz    1 (28%)       1 (28%)     1G (13%)         1G (13%)       27m
+----
+
+. Verify that the `cgroups` are set up correctly. Get the process ID (PID) of the `pause` process by running the following command:
++
+[source,terminal]
+----
+# systemctl status | grep -B5 pause
```

Comment:
this may have unstable results... 

oc describe pod/cpumanager | grep slice
then systemctl status | grep slice-name

Reply:
This actually was thought to be run on worker node. Current documentation doesn't say how to get PID of the pause process which is running in container. Although it is showing the tree output from worker node. 
 Silke we can write here that we need to run debug pod on worker node and then do systemctl status..
 Paul do you know of any other method?